### PR TITLE
Use different library to enable support for UnifiOS based devices

### PR DESCRIPTION
**Pull request by tickez**
In case UnifiOS support is desired, this should make it work. 
Please note that the library used seems not really supported any longer, but the same seems true for unifi-events. 
I added the dependency to my github fork for the time being and raised a PR with all the changes to the original maintainer of ubnt-unifi.

### CODE REVIEW

**Comment by Kienz** [comment]
Can this be merged?

File: package.json [hunk 1]
```diff
@@ -19,6 +19,6 @@
     "homebridge-plugin"
   ],
   "dependencies": {
-    "unifi-events": "^0.4.3"
+    "ubnt-unifi": "tickez/ubnt-unifi"
```
Comment (oznu) [inline package.json:22]: This needs to be a depedency on npm.

Can you create a pull request against unifi-events with the changes from ubnt-unifi?  (ubnt-unifi is a fork of unifi-events).

**Comment by oznu** [comment]
Thanks for this, the changes is published under the `@test` tag. Can you confirm all is working as expected?

**Comment by oznu** [comment]
https://github.com/homebridge/homebridge/wiki/How-to-Install-Alternate-Plugin-Versions

**Comment by tickez** [comment]
Verified to work for me with my standalone controller. Someone with a UnifiOS based controller should confirm too.

**Comment by Kienz** [comment]
I'm using UDM-Pro with UniFiOS and it doesn't work.
Log-Message:
[1/27/2021, 12:00:28 PM] [Stefan Anwesend] Getting list of connected clients from UniFi Controller...
[1/27/2021, 12:00:33 PM] [Stefan Anwesend] ERROR: Failed to check occupancy: 401 - "Unauthorized"

Config:
```json
{
    "accessory": "UniFi Occupancy Sensor",
    "name": "Stefan Anwesend",
    "unifi": {
        "controller": "https://192.168.1.1/",
        "username": "homebridge",
        "password": ",xxxxxxxxxb?9",
        "site": "Default",
        "secure": false,
        "unifios": true
    },
    "watch": [
        "xx:xx:xx:42:xx:79"
    ],
    "mode": "all"
}
```

If I made the REST Api requests by my own it works:
POST https://192.168.1.1:443/api/auth/login (using same username/password in the body)
=> Login successful
The getting the list of connected devices:
GET https://192.168.1.1:443/proxy/network/api/s/default/stat/sta (Using Cookie TOKEN in the header).
=> it returns the JSON

**Comment by Kienz** [comment]
I think this is helpful:
https://github.com/josiasmontag/unifi-events/commit/c3cf0a2b03482e0bb5c9aab8e6ecb83dad71f992

The URL is on UniFiOS different (api/self => api/users/self)
```js
  _ensureLoggedIn () {
    return this.rp.get(`${this.controller.href}api/users/self`)
      .catch(() => {
        return this._login()
      })
}
```

**Comment by oznu** [comment]
pushed `@beta.2`

**Comment by tickez** [comment]
`@test.2` still good for me

**Comment by Kienz** [comment]
Works with UniFiOS on UDM-Pro.

**Comment by oznu** [comment]
Thanks all. Publish to latest.

**Pull request merged**